contracts/truefi2/SAFU.sol

```diff
@@ -15,9 +15,27 @@ contract SAFU is UpgradeableClaimable {
     using SafeMath for uint256;
     using SafeERC20 for IERC20;
 
+    // ================ WARNING ==================
+    // ===== THIS CONTRACT IS INITIALIZABLE ======
+    // === STORAGE VARIABLES ARE DECLARED BELOW ==
+    // REMOVAL OR REORDER OF VARIABLES WILL RESULT
+    // ========= IN STORAGE CORRUPTION ===========
+
     ILoanFactory2 public loanFactory;
     ILiquidator2 public liquidator;
 
+    mapping(ILoanToken2 => uint256) public loanDeficit;
```

Comment:
Not sure if we need this mapping since we plan to use ERC20 for tracking deficit anyway so it'll get removed